client/client.go

```diff
@@ -67,24 +68,30 @@ type ChargePayload struct {
 
 // Offer is the api message of an Offer.
 type Offer struct {
-	ID string
+	ID int
 
-	OfferType OfferType
-	// If OfferType is not Instance, InstanceID would be empty, otherwise InstanceID may be the sku of that instance.
 	// e.g. AWS: 9QH3PUGXCYKNCYPB, GCP: 0009-6F35-3126
-	InstanceID string
+	InstanceSKU string
```

Comment:
Seems like this should be inside InstancePayload.

Reply:
InstanceSKU functions as a foreign key here.